test/e2e/config/e2e_conf.yaml

```diff
@@ -44,6 +44,23 @@ providers:
           new: "--leader-elect=false"
         - old: --metrics-addr=127.0.0.1:8080
           new: --metrics-addr=:8080
+  - name: docker-updated
```

Comment:
I would be in favor of not duplicating this since it is a one liner less compared to the original cluster template we have in the repo, however, it looks like bigger of a change to achieve this (using jinja templating or Go's templating capabilites) vs benefits it brings.